pkg/inst/tests/test_rdd.R

```diff
@@ -314,6 +314,18 @@ test_that("values() on RDDs", {
   expect_equal(actual, lapply(intPairs, function(x) { x[[2]] }))
 })
 
+test_that("pipeRDD() on RDDs", {
+  actual <- collect(pipeRDD(rdd, "cat"))
```

Comment:
"more" works on Windows. I think "more" is better than "cat" beause OS type check can be avoided.